bootstrap/v2/pkg/kfapp/existing_arrikto/existing.go

```diff
@@ -154,13 +155,23 @@ func (existing *Existing) Apply(resources kftypes.ResourceEnum) error {
 		KubeflowEndpoint        string
 		OIDCEndpoint            string
 		AuthServiceClientSecret string
+		AuthServiceHmacSecret   string
+		OIDCRedirectUris        []string
 		KubeflowUser            *kfUser
 	}{
 		KubeflowEndpoint:        kfEndpoint,
 		OIDCEndpoint:            oidcEndpoint,
-		AuthServiceClientSecret: genRandomString(32),
+		AuthServiceClientSecret: base64.StdEncoding.EncodeToString([]byte(genRandomString(32))),
+		AuthServiceHmacSecret:   base64.StdEncoding.EncodeToString([]byte(initialiseHMACSecretFromEnvOrGen("JWT_HMAC_SECRET", 64))),
```

Comment:
I don't believe it's necessary to copy the whole logic.
Let's just call genRandomString immediately.